src/test/java/com/mercateo/common/rest/schemagen/link/LinkCreatorTest.java

```diff
@@ -190,7 +200,7 @@ protected Something getReturnType(ImplementedBeanParamType param) {
     }
 
     @Test
-    public void testTargetSchemaPresentOnMatchingMediaTypeAtMethodLevel()
+    public void testTargetSchemaPresentOnMatchingMediaTypeAtMethodLevel__targetSchemaEnabled()
```

Comment:
Underscore in method name does not match the naming scheme of the other test methods

Reply:
done